services/general-routing/generalrouting-impl/pom.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>general-routing</artifactId>
```

Comment:
Maybe, I misunderstood the architecture of application. We have one main router(module "general-routing"). It parses all our request by type. If type is "authorization" we delegate our request to AuthorizationRouter, that then delegates this request to one of handlers. If type is "profile" we delegate our request to ProfileRouter. I don't have other idea, how to make routing by type without main router. Now I see such hierarchy: (main_router) -> (AuthRouter, ProfileRouter, MessengerRouter) - > (A lot of handlers...). I also see the variant to make main routing later in rest contollers. What variant is right?